.env.development

```diff
@@ -6,7 +6,7 @@ DB_PORT=3306
 DB_HOST='localhost'
 DB_USER='bhima'
 DB_PASS='HISCongo2013'
-DB_NAME='bhima_test'
+DB_NAME='imck'
```

Comment:
This should be reverted to "bhima_test".

Reply:
okay

Reply:
This can't be merged until this is fixed.